include/picongpu/plugins/common/openPMDVersion.def

```diff
@@ -29,6 +29,26 @@ namespace picongpu
 {
     namespace openPMD
     {
+        /** PIConGPU's IO major file version.
+         *
+         * The version can be used to handle incompatibilities between checkpoint files in case breaking changes
+         * within the openPMD checkpoint code are introduced.
+         * A change in the major version points to a new feature/fix which can not be handled by older PIConGPU IO
+         * implementations. Newer PIConGPU IO implementations can optionally be support old major versions.
+         *
+         * @attention If the version is changed please update openPMDWriter::checkIOFileVersionCompatibility().
+         */
+        static constexpr int picongpuIOVersionMajor = 0;
+
+        /** PIConGPU's IO minor file version.
+         *
+         * A change in the minor version means that the new introduced feature/fix can be loaded by all PIConGPU IO
+         * implementations with the same major IO file version.
+         *
+         * @attention If the version is changed please update openPMDWriter::checkIOFileVersionCompatibility() if
+         * needed.
+         */
+        static constexpr int picongpuIOVersionMinor = 0;
```

Comment:
Can we start with version 0.1 ? As 0.0 may be confused as not a valid version number (so a user may think the version information itself is incorrect).

Reply:
Possible.

Reply:
done